tests/test_data.py

```diff
@@ -127,14 +128,15 @@ def test_seed(self):
 
 
 class TestRandomSystem:
-    def test_sizes_and_dimensions(self):
-        for N in (0, 1, 10, 100, 1000):
-            for is2D in (True, False):
-                box, points = freud.data.make_random_system(
-                    box_size=10, num_points=N, is2D=is2D
-                )
-                assert points.shape == (N, 3)
-                assert box.is2D == is2D
+    @pytest.mark.parametrize(
+        "N, is2D", [(N, is2D) for N in (0, 1, 10, 100, 1000) for is2D in (True, False)]
+    )
```

Comment:
Separate these parametrizations:
```suggestion
    @pytest.mark.parametrize("N", (0, 1, 10, 100, 1000))
    @pytest.mark.parametrize("is2D", (True, False))
```